packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -859,6 +868,10 @@ class FlutterTask extends BaseFlutterTask {
         return intermediateDir
     }
 
+    String getAssetsDirectory() {
+        return "${outputDirectory}/flutter_assets/"
```

Comment:
we should remove the trailing / as it's not needed for chmod and with it we will have 2 slashes in the windows command above.